docs/BUILDING.md

```diff
@@ -172,32 +170,27 @@ _Note: If you're using Visual Studio Code, the [cpack plugin](https://marketplac
 # Clone the repo
 git clone https://github.com/HarbourMasters/Shipwright.git
 cd ShipWright
+
 # Clone the submodule libultraship
 git submodule update --init
-# Copy the baserom to the OTRExporter folder
-cp <path to your ROM> OTRExporter
+
 # Generate Ninja project
-cmake -H. -Bbuild-cmake -GNinja # -DCMAKE_BUILD_TYPE:STRING=Release (if you're packaging)
-# Extract assets & generate OTR (run this anytime you need to regenerate OTR)
-cmake --build build-cmake --target ExtractAssets
+# Add `-DCMAKE_BUILD_TYPE:STRING=Release` if you're packaging
+cmake -H. -Bbuild-cmake -GNinja
+
+# If you need a newer soh.otr only
+cmake --build build-cmake --target GenerateSohOtr
+
 # Compile the project
-cmake --build build-cmake # --config Release (if you're packaging)
+# Add `--config Release` if you're packaging
+cmake --build build-cmake
 
 # Copy oot.otr into the Application Support directory
 cp build-cmake/soh/oot.otr ~/Library/Application\ Support/com.shipofharkinian.soh/
```

Comment:
While you are in here, we can drop this now as its not required for local dev on Mac.
```suggestion
```